README.md

```diff
@@ -1 +1,52 @@
 # saucelabs-connector
+
+Helps connect the local machine to SauceLabs and starts a browser with the specified page url.
+
+##Install
+
+`$ npm install saucelabs-connector`
+
+##Usage
+```js
+
+// Configure browsers here: https://docs.saucelabs.com/reference/platforms-configurator/
```

Comment:
the link is obsolete. Now it's redirecting to `https://wiki.saucelabs.com/`, please update to the actual link